manifests/init.pp

```diff
@@ -105,6 +105,11 @@
 #                               try to determine this automatically.
 #                               type:boolean
 #
+# $templates::                  Use Templates
+#                               type:boolean
+#
+# $template_url::               URL a host should use for provisioning templates
```

Comment:
Host might be ambigous in the content of a proxy. Maybe explicitly use client if you mean hosts that are provisioning?